tests/tasks/remove_candlepin_container.yml

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Check if the candlepin container exists
+  command:
+    argv:
+      - podman
+      - ps
+      - -a
+      - --filter
+      - name=candlepin
+  register: __rhc_candlepin_cont_exists
+  changed_when: false
+
+- name: Stop and remove Candlepin container
+  command:
+    argv:
+      - podman
+      - stop
+      - candlepin
+  changed_when: true
+  when:
+    - lsr_rhc_test_data_file | length == 0
```

Comment:
Rather than checking this only for a specific task, better apply a whole `block` for all the tasks, mimicking what `setup_candlepin.yml` does. This way:
- this task file can be imported without checking what's the testing configuration (similar to `setup_candlepin.yml`)
- in case more tasks need to be done for stopping candlepin, they can be added in the existing block